src/commands/compareReports.js

```diff
@@ -108,7 +108,13 @@ export default async function compareReports(
   // Make second compare call to finalize the deep compare. The second call will
   // cause a status to be posted to the PR (if applicable). Any ignored diffs
   // from the first call will be excluded from the result.
-  const secondCompareResult = await makeCompareCall(false);
   log(secondCompareResult.summary);
+
+
+  const secondCompareResult = await makeCompareCall(false);
+  if(afterComparison && typeof afterComparison === "function") {
+    afterComparison(secondCompareResult);
```

Comment:
Make sure to add the callback on line 52 as well. That's the more common case, right here is where there is a `compareThreshold` used. Which is rare.

Reply:
I added also condition for isAsync